app/views/users/_user.json.jbuilder

```diff
@@ -0,0 +1,2 @@
+json.extract! user, :id, :name, :lastName, :birthday, :weight, :height, :email, :created_at, :updated_at
+json.url user_url(user, format: :json)
```

Comment:
If you don't need all the json responses, remove all the corresponding code and files